vulnerabilities/importers/apache_httpd.py

```diff
@@ -49,47 +62,110 @@ def updated_advisories(self):
         # skips processing the response further to avoid duplicate work
 
         if create_etag(data_src=self, url=self.url, etag_key="ETag"):
-            data = fetch_xml(self.url)
-            advisories = to_advisories(data)
+            links = fetch_links(self.url)
+            self.set_api()
+            advisories = []
+            for link in links:
+                data = requests.get(link).json()
+                advisories.append(self.to_advisory(data))
             return self.batch_advisories(advisories)
 
         return []
 
+    def to_advisory(self, data):
+        cve = data["CVE_data_meta"]["ID"]
+        descriptions = data.get("description", {}).get("description_data", [])
+        description = None
+        for desc in descriptions:
+            if desc.get("lang") == "eng":
```

Comment:
When does this condition not get satisifed, ie does `desc.get("lang") == None` happen ?

Reply:
No

Reply:
I'm guessing that `desc.get("lang")` is never `None` (`"lang"` key is always present in `desc`) . If yes then use `if desc["lang"] == "eng" : .....`